egui_glow/src/painter.rs

```diff
@@ -268,16 +196,26 @@ impl Painter {
         if self.egui_texture_version == Some(texture.version) {
             return; // No change
         }
-
+        let gamma = if self.post_process.is_none() {
+            1.0 / 2.2
```

Comment:
The glow painter on `main` doesn't need a post-processing step, because the native framebuffer is configured with linear (sRGB-aware) blending. So the gamma should be `1.0` unless we are on web and the post-processing step is missing.

```suggestion
        let gamma = if self.is_web && self.post_process.is_none() {
            1.0 / 2.2
```

…unless I'm missing something